src/env.cc

```diff
@@ -364,7 +374,20 @@ bool Environment::RemovePromiseHook(promise_hook_func fn, void* arg) {
 void Environment::EnvPromiseHook(v8::PromiseHookType type,
                                  v8::Local<v8::Promise> promise,
                                  v8::Local<v8::Value> parent) {
-  Environment* env = Environment::GetCurrent(promise->CreationContext());
+  Local<v8::Context> context = promise->CreationContext();
+
+  // Grow the embedder data if necessary to make sure we are not out of bounds
+  // when reading the magic number.
+  context->SetAlignedPointerInEmbedderData(ContextEmbedderIndex::kContextTagBoundary, nullptr);
+  int* magicNumberPtr = reinterpret_cast<int*>(
+    context->GetAlignedPointerFromEmbedderData(ContextEmbedderIndex::kContextTag)
```

Comment:
nit: four space indentation. You can move the ) on the next line to this line.

Reply:
done (the parenthesis part, the indent used looks like 2 spaces to me? or is 4 a special rule for these cases?)